src/neo/Cryptography/MPT/MPTTrie.Delete.cs

```diff
@@ -82,14 +86,15 @@ private bool TryDelete(ref MPTNode node, ReadOnlySpan<byte> path)
                         }
                         if (lastChild is HashNode hashNode)
                         {
-                            lastChild = Resolve(hashNode);
-                            if (lastChild is null) return false;
+                            lastChild = Resolve(hashNode.Hash);
+                            if (lastChild is null) throw new System.ArgumentNullException("Invalid hash node");
```

Comment:
```suggestion
                            if (lastChild is null) return false;
```

Reply:
Done. We should throw exception. It's a structure error. I change the exception to InvalidOperationException.